index.js

```diff
@@ -130,13 +130,21 @@ program
   .option('-f, --force-reload', 'force a full reload of all site files when a file is changed')
   .option('-p, --port <port>', 'port for server to listen on (Default is 8080)')
   .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .option('--one-page <file>', 'render and serve only a single page in the site')
   .option('--no-open', 'do not automatically open the site in browser')
   .action((root, options) => {
     const rootFolder = path.resolve(root || process.cwd());
     const logsFolder = path.join(rootFolder, '_markbind/logs');
     const outputFolder = path.join(rootFolder, '_site');
+    const forceReload = options.onePage || options.forceReload;
 
-    const site = new Site(rootFolder, outputFolder, options.forceReload, options.siteConfig);
+    if (options.onePage) {
+      // replace slashes for paths on Windows
+      // eslint-disable-next-line no-param-reassign
+      options.onePage = options.onePage.replace('\\', '/');
```

Comment:
On Windows nodejs supports both `/` and `\`. Is there any particular reason why this was done?

Reply:
On PowerShell and `cmd.exe`, paths are normally written with `\`. However, we internally normalise all paths to use `/` as the separator (including when storing the value of `pages.src` in `siteData.json`). Since the value passed to `--one-path` is directly used to look up which page to serve, I believe it would be better to allow users to enter `\` as the path separator when specifying the file path to `--one-page`, which would be less confusing than if an error was emitted when `\` is used.

Reply:
Replacing `options.onePage.replace('\\', '/')` with `path.normalize()` actually breaks the lookup. This is because the default `site.json` uses paths with `/`. While we use `path.` functions for filesystem operations, `page.src` is still written with `/` into `siteData.json`. 

`path.normalize()` on a Windows system converts all path seperators to `\`, which breaks the lookup since pages are located by `page.src`, which needs `/` to match.

Reply:
How about `path.posix.normalize`?

Reply:
@acjh thank you for the suggestion! Unfortunately, `path.posix.normalize` on Windows-style paths does nothing:
```js
> path.posix.normalize('userGuide\\userQuickStart.md')
'userGuide\\userQuickStart.md'
```
This is because, according to the [Node.js docs for `path.normalize`](https://nodejs.org/docs/latest-v8.x/api/path.html#path_path_normalize_path), only `/` is considered a path separator under POSIX, and thus `\` will not be replaced. 

(The phrasing is clearer in the original PR for this section nodejs/node#12700, but was subsequently changed.)

Reply:
Understood. Let's add a function in utils similar to ensure-posix-path's [`ensurePosix`](https://github.com/stefanpenner/ensure-posix-path/blob/master/index.js).

Reply:
@acjh Sure, that will work!